### PR TITLE
doc(1.8.0): add important note with things to check before upgrade

### PR DESCRIPTION
**Pull request by james-munson**
#### Which issue(s) this PR fixes:

Issue https://github.com/longhorn/longhorn/issues/9440

#### What this PR does / why we need it:

Mention some important items to check before doing upgrade.

#### Special notes for your reviewer:

When the text is approved, this will be copied to v1.6.x and v1.7.x docs.

### CODE REVIEW

**Comment by derekbit** [comment]
@james-munson 
Also need to add it to upgrade page and v1.7.x and v1.6.x.

**Comment by james-munson** [comment]
> @james-munson Also need to add it to upgrade page and v1.7.x and v1.6.x.

Right.  If we're good with the content, I will go ahead and replicate it.

**Review by jillian-maroket — COMMENTED** [review]
Review done

File: content/docs/1.8.0/important-notes/_index.md [hunk 1]
```diff
@@ -35,6 +36,12 @@ The functionality of the [environment check script](https://github.com/longhorn/
 ### Upgrade Check Events
 Longhorn performs a pre-upgrade check when upgrading with Helm or Rancher App Marketplace.  If a check fails, the upgrade will stop and the reason for the check's failure will be recorded in an event.  For more detail, see [Upgrading Longhorn Manager](../deploy/upgrade/longhorn-manager).
 
+### Manual Checks
+Not every upgrade path performs the automated checks, and some situations may not be covered by the tool.  There are also some things it is recommended to check manually, via kubectl or the UI, and either take some action or defer the upgrade until they can be fixed:
```
[the remark this answers is not in the document]
Comment (james-munson) [reply content/docs/1.8.0/important-notes/_index.md]: Done, thanks.

File: content/docs/1.8.0/important-notes/_index.md [hunk 2]
```diff
@@ -35,6 +36,12 @@ The functionality of the [environment check script](https://github.com/longhorn/
...
+- Ensure that all V2 engine volumes are detached and the replicas are stopped.  V2 does not yet support live upgrade.
+- Don't upgrade with a volume in "Faulted" status.  If all the replicas are deemed unusable, it is possible that they may be deleted, leading to loss of data if there is no usable backup to restore.
+- Likewise, avoid upgrading if there is a failed BackingImage.  See [Backing Image](../advanced-resources/backing-image/backing-image).
```
[the remark this answers is not in the document]
Comment (james-munson) [reply content/docs/1.8.0/important-notes/_index.md]: Done, thanks.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**